dist/all.js

```diff
@@ -332,6 +332,25 @@ RiseVision.Common.Utilities = (function() {
     return div.textContent;
   }
 
+  function hasInternetConnection(filePath, callback) {
+    var xhr = new XMLHttpRequest();
+
+    if (!filePath || !callback || typeof callback !== "function") {
+      return;
+    }
+
+    xhr.open("HEAD", filePath + "?cb=" + new Date().getTime(), false);
+
+    try {
+      xhr.send();
+
+      callback((xhr.status >= 200 && xhr.status < 304));
```

Comment:
@rodrigopavezi The request will always include a cachebuster, so I'm not sure if allowing 304 is right.